web/modules/custom/dpl_cache_settings/dpl_cache_settings.module

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @file
+ * Functionality for DPL cache settings.
+ */
+
+use Drupal\Core\Url;
+
+/**
+ * Implements hook_toolbar_alter().
+ *
+ * Displaying the DPL CMS version in the secondary admin toolbar.
+ *
+ * @param array<mixed> $items
+ *   See hook_toolbar_alter().
+ */
+function dpl_cache_settings_toolbar_alter(array &$items): void {
+  $items['cache_clear'] = [
+    '#type' => 'toolbar_item',
+    'tab' => [
```

Comment:
I do not think it is a good idea to place access to this item in the secondary admin toolbar.

Clearing the caches it not something that we want administrators to do. We consider it a problem with our code if that is necessary. Putting it in the toolbar puts the option within plain sight. It takes fewer clicks to go to the clear cache form than the create article form.

Administrators already use the settings part of the administration interface so I do not see a reason why we cannot put our own custom form there as well.

Reply:
Any suggestions where to put it?
I get your point, but, I would imagine DDF is gonna get spammed when 85 libraries suddenly write, asking where the cache clear button is.

I've done stuff to make the cache clear page 'scary' - I've added the long warning text, and made the button red, like delete buttons.

Reply:
I suggest placing the form as an item under `/admin/config/system`.

I do not think we should be afraid that the functionality is placed within the administration settings:

1. We do not want or expect clearing the cache to be necessary
2. The functionality was also placed within administration settings in previous versions of the system

Reply:
I've moved it away as you suggested